metrics-exporter-prometheus/src/recorder.rs

```diff
@@ -273,4 +287,10 @@ impl PrometheusHandle {
     pub fn render(&self) -> String {
         self.inner.render()
     }
+
+    /// Purges registry's histogram data by draining it into the distribution. This should be
+    /// called periodically to prevent the accumulation of histogram samples.
+    pub fn purge(&self) {
```

Comment:
I think we should just change the wording overall from "purge" to "upkeep", and make this `run_upkeep`.

Purging to me sounds like getting rid of, when realistically we're just doing periodic cleanup work.